airflow/www/views.py

```diff
@@ -1842,7 +1847,8 @@ class GraphForm(DateTimeWithNumRunsWithDagRunsForm):
             nodes=nodes,
             edges=edges,
             show_external_log_redirect=task_log_reader.supports_external_link,
-            external_log_name=external_log_name)
+            external_log_name=external_log_name,
```

Comment:
I was mostly thinking of the current DagRun.

You can get the state of it already using `dt_nr_dr_data['dr_state']` (also used on L1839)

Reply:
Fixed, thanks @kaxil!